app_dart/lib/src/model/luci/buildbucket.dart

```diff
@@ -207,6 +207,164 @@ class GetBuildRequest extends JsonBody {
   }
 }
 
+/// A request for the GetBuilder RPC.
+@JsonSerializable(includeIfNull: false)
+class GetBuilderRequest extends JsonBody {
```

Comment:
Is this used anywhere?

Reply:
No at this moment. Added it here for potential future support.